ansible_collections/infoblox/nios_modules/galaxy.yml

```diff
@@ -0,0 +1,62 @@
+### REQUIRED
+
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: infoblox
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: nios_modules
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 1.0.0
+
+# The path to the Markdown (.md) readme file. This path is relative to the root of the collection
+readme: README.md
+
+# A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
+# @nicks:irc/im.site#channel'
+authors:
+- Sailesh Giri (sgiri@infoblox.com)
+- Vaishnavi TR (vtr@infoblox.com)
+- Anagha KH (akh@infoblox.com)
+- Shankar Ganesh (sganesh@infoblox.com)
+
+### OPTIONAL but strongly recommended
+
+# A short summary description of the collection
+description: Infoblox Ansible Collection for vNIOS allows managing your NIOS objects through APIs.
+
+# Either a single license or a list of licenses for content inside of a collection. Ansible Galaxy currently only
+# accepts L(SPDX,https://spdx.org/licenses/) licenses. This key is mutually exclusive with 'license_file'
+license:
+- GPL-3.0-only
+
+# The path to the license file for the collection. This path is relative to the root of the collection. This key is
+# mutually exclusive with 'license'
+license_file: ''
+
+# A list of tags you want to associate with the collection for indexing/searching. A tag name has the same character
+# requirements as 'namespace' and 'name'
+tags:
+- infoblox
+- nios
+
+# Collections that this collection requires to be installed for it to be usable. The key of the dict is the
+# collection label 'namespace.name'. The value is a version range
+# L(specifiers,https://python-semanticversion.readthedocs.io/en/latest/#requirement-specification). Multiple version
+# range specifiers can be set and are separated by ','
+dependencies: 
+- infoblox-client
```

Comment:
this syntax is incorrect apparently, yields

```
AttributeError: 'list' object has no attribute 'items'
```

Reply:
Could you please let me know what is the right syntax?

Reply:
I just removed the whol
```
dependencies: 
- infoblox-client
```
block. From the comment above, it seems that this dependencies block pertain only to the "collections" you depend on, not the rest of the pip dependencies... personally I still maintain a requirements.yml for collections + a requirements.txt for pip dependencies.

Reply:
Sure I will also remove the same. Thank you Freedge

Reply:
Thanks for your work! one last thing (just mentionning this here) the lookup does not seem performed through the collection. I use this:
```
  collections:
    - infoblox.nios_modules
...
    set_fact:
      lease: "{{ lookup('nios', 'lease', filter=nios_filter, provider=nios_provider, return_fields=nios_fields ) }}"
```

and this does not seem to be calling any code from the collection (unlike for example, calling "nios_fixed_address" that does indeed, uses the code from the collection. Do you know if there is something to do to use the collection for the lookup?